bcda/client/bluebutton.go

```diff
@@ -194,3 +205,12 @@ func GetDefaultParams() (params url.Values) {
 	params.Set("_format", "application/fhir+json")
 	return params
 }
+
+func HashHICN(toHash string) (hashedValue string) {
+	pepper, err := hex.DecodeString("b8ebdcc47fdd852b8b0201835c6273a9177806e84f2d9dc4f7ecaff08681e86d74195c6aef2db06d3d44c9d0b8f93c3e6c43d90724b605ac12585b9ab5ee9c3f00d5c0d284e6b8e49d502415c601c28930637b58fdca72476e31c22ad0f24ecd761020d6a4bcd471f0db421d21983c0def1b66a49a230f85f93097e9a9a8e0a4f4f0add775213cbf9ecfc1a6024cb021bd1ed5f4981a4498f294cca51d3939dfd9e6a1045350ddde7b6d791b4d3b884ee890d4c401ef97b46d1e57d40efe5737248dd0c4cec29c23c787231c4346cab9bb973f140a32abaa0a2bd5c0b91162f8d2a7c9d3347aafc76adbbd90ec5bfe617a3584e94bc31047e3bb6850477219a9")
```

Comment:
If this needs to work in production with an actually secret pepper, we probably need a different mechanism for passing the pepper.

Reply:
I bow to the explanation of the use case given out of band.

Reply:
Just going to post the convo here for everyone's benefit.

This value came from a confluence page and is duplicated in 2 gists linked from that page.  The point of this process is to define how a HICN should be hashed so that external partners and BlueButton are aligned.  Comparing an externally generated hash to an internally generated one requires that both parties follow the same recipe or process.  I believe that this necessitates exposing this value publicly in the same way one might expose a public key.

I am happy to keep this convo going here or on slack.